docs/configuration/validation/brk.rst

```diff
@@ -0,0 +1,43 @@
+.. _configuration_validation_brk:
+
+=========================
+BRK - Zakelijk gerechtigd
+=========================
+
+The special *AddressNL* component has a specific validator available, that can
+check if the currently logged in user is the business owner of the filled-in property.
+
+Configuration
+=============
+
+1. Obtain a BRK API key.
+2. In Open Forms, navigate to: **Miscellaneous** > **KvK configuration**.
```

Comment:
we tend to point people to the configuration overview page (the Misc menu entry should not be used directly) and have them click the respective configure option there. I see that the [KVK basisprofiel](https://open-forms.readthedocs.io/en/latest/configuration/prefill/kvk.html#configuration-prefill-kvk) documentation is also wrong in that regard.

A good example is: https://open-forms.readthedocs.io/en/latest/configuration/prefill/bag.html